docs/extendedusage.rst

```diff
@@ -198,6 +200,23 @@ Traits
     'The index of the first track to return'
 
 
+RAML1.0 Types
+~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    >>> api.types
+    {'Person': ObjectType(name='Person', properties={'name': Property(type='string')})}
```

Comment:
Hmm but I'm not sure a dictionary is appropriate. The rest of the library returns a list (which preserves order) of the relevant objects, like `api.resources` and `resource.query_params` etc.